components/ExpenseReporter/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react'
+import { Bar, BarChart, CartesianGrid, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
+
+function ExpenseReporter() {
+  const [expenseValue, setValues] = useState([])
+
+  const today = new Date()
+  const lastSixMonth = new Date(today.getTime() - 6 * 31 * 24 * 60 * 60)
+
+  const urlLink = `?startmonth=${today.getFullYear()}${today.getMonth()}&endmonth=${lastSixMonth.getFullYear()}${lastSixMonth.getMonth()}`
+
+  useEffect(() => {
+    fetch(`${process.env.WALLORA_BACKEND_TOTAL_PLANNED_AMOUNT_WORKITEMWISE}${urlLink}`)
+      .then((response) => response.json())
+      .then(({ data: expenses }) => {
+        setValues(expenses)
+      })
+  }, [])
+
+  return (
+    <div>
+      <div className="text-center">Major Expenses planned in next 6 months</div>
+      <br />
+      <ResponsiveContainer width="95%" height={400} className="text-center">
+        <BarChart width={1600} height={400} data={expenseValue} className="ml-8">
+          <CartesianGrid strokeDasharray="3 3" />
+          <XAxis dataKey="name" />
+          <YAxis />
+          <Tooltip />
+          <Legend />
+          <Bar dataKey="amount" fill="#ffc1c1" />
```

Comment:
use color from material ui theme. Move this color code there

Reply:
this doesn't use material ui components hence won't help moving it there. Tried adding it in tailwind config and using since fill is not className thatdoesn't work